.travis.yml

```diff
@@ -1,23 +1,30 @@
 language: scala
 scala:
-  - 2.11.12
+  - 2.13.5
 jdk:
   - openjdk8
   - openjdk11
 
+before_script:
+  # Download sbt because Travis can't find it automatically :(
+  - mkdir -p $HOME/.sbt/launchers/1.5.0/
+  - curl -L -o $HOME/.sbt/launchers/1.5.0/sbt-launch.jar https://repo1.maven.org/maven2/org/scala-sbt/sbt-launch/1.5.0/sbt-launch-1.5.0.jar
+
 script:
-  - sbt clean coverage +test coverageReport && sbt coverageAggregate
-after_success:
-  # Upload coverage reports to codecov.io
-  - bash <(curl -s https://codecov.io/bash) -t 50d829ce-40d0-4fa4-8479-2e0f3864a07f
+  - sbt clean coverage +test coverageAggregate
+  # Upload coverage reports to codecov.io before checking binary compatibility
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
I believe that there's a cached version in the global pipeline library to which we should probably shift.

(maybe put as a todo?)

Reply:
Unfortunately, those scripts are only cached on Rally's internal Artifactory